docs/configuration/tips.mdx

```diff
@@ -70,6 +76,8 @@ If you would like your nodes to include/expand the "public" mesh, you must use t
 
 It is **strongly** recommended to leave your [MAX HOPS](/docs/configuration/radio/lora#max-hops) set to 3 unless you're sure you need more (or less) to reach your destination node. Unnecessarily high hop counts often lead to network issues.
 
+![Hop Count Image](/img/configuration/max-hops.webp)
```

Comment:
Does this file already exist? It's not being added in your PR

Reply:
Yes, it's used elsewhere:
https://meshtastic.org/docs/configuration/radio/lora/#max-hops